src/main/java/frc/robot/RobotContainer.java

```diff
@@ -248,58 +249,58 @@ private void configureDriveBindings() {
      * Configures button bindings for the roller / elevator mechs and controller.
      */
     private void configureMechBindings() {
-        mechAButton.onTrue(new DropperChooserCommand(driveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
+        //mechAButton.onTrue(new DropperChooserCommand(driveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
 
         rollerSubsystem.setDefaultCommand(new RunCommand(() -> {
```

Comment:
maybe we can break roller & elevator bindings into seperate functions so we can just comment out `configureElevatorBindings()` in the future.